markdown/events/2022/bytesize-resources-to-learn-nextflow.md

```diff
@@ -23,3 +23,59 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[3:45](https://www.youtube.com/watch?v=oO7rAp-QZOkt=225)
+The core features of Nextflow that should interest bioinformations is it enables workflow portability and reproducibility, simplify parallelization and large scale deployment, easily integrates existing tools, systems and industry standards. These are just some of the features. I have here linked two video links that give an overview of nf-core and Nextflow and also some of the details of why nf-core or Nextflow stands out in comparison to other workflow management systems. I would recommend if you're a complete beginner to go over these videos as well. But once you're through, we are talking about you might already have an introduction to Nextflow. But I want to like completely go through the nuts and bolts of to learn the whole syntax.
+
+[4:53](https://www.youtube.com/watch?v=oO7rAp-QZOkt=)
+We are recommending the courses for you here. Because now DSL2 syntax is default, we would like to transition most of the courses to DSL2 syntax as well. In that aspect, we are recommending three courses here. One is the Nextflow tutorial variant calling edition that was developed by myself and colleagues at CDC. This particular workshop content was developed so that it explains the concepts through a variant calling workflow example. I'll briefly walk you through the contents on how it has been divided for this particular workshop that we have developed. It starts off with an introduction to Nextflow, and then we dive right into introduction to nf-core with the main aspect being that we wanted to tell participants not to reinvent the wheel in writing pipelines. And an introduction to nf-core is also a great way of introducing how to run Nextflow pipelines straight out of the box, and also to show how they can be done on local on-prem HPC clusters. We introduce to these sessions here, and then we dive into the details or start with the syntax of Nextflow, then get into the details of channels, processes, workflows, and operators. And finally, we give a challenge of converting a bash script of a variant calling workflow into Nextflow processes. And finally, we show the workflow. The content will progress into modularizing all of the processes that are involved in the variant calling workflow and towards the end is where we have introduced a chapter on assembling the variant calling workflow now using nf-core modules. Previously we show how you can develop your own local modules, but we also show how the same process can be done using nf-core modules. It comes as a full circle towards the end. This session, we have designed for nine hours and it has worked out pretty well in training.
```

Comment:
```suggestion
We are recommending thrse courses for you here. Because now DSL2 syntax is default, we would like to transition most of the courses to DSL2 syntax as well. In that aspect, we are recommending three courses here. One is the Nextflow tutorial variant calling edition that was developed by myself and colleagues at CDC. This particular workshop content was developed so that it explains the concepts through a variant calling workflow example. I'll briefly walk you through the contents on how it has been divided for this particular workshop that we have developed. It starts off with an introduction to Nextflow, and then we dive right into introduction to nf-core with the main aspect being that we wanted to tell participants not to reinvent the wheel in writing pipelines. And an introduction to nf-core is also a great way of introducing how to run Nextflow pipelines straight out of the box, and also to show how they can be done on local & on-prem HPC clusters. We introduce to these sessions here, and then we dive into the details or start with the syntax of Nextflow, then get into the details of channels, processes, workflows, and operators. And finally, we give a challenge of converting a bash script of a variant calling workflow into Nextflow processes. And finally, we show the workflow. Then content will progress into modularizing all of the processes that are involved in the variant calling workflow and towards the end is where we have introduced a chapter on assembling the variant calling workflow now using nf-core modules. Previously we show how you can develop your own local modules, but we also show how the same process can be done using nf-core modules. It comes as a full circle towards the end. This session, we have designed for nine hours and it has worked out pretty well in training.
```